src/seedu/addressbook/AddressBook.java

```diff
@@ -257,18 +257,17 @@ private static void echoUserCommand(String userCommand) {
      * @param args full program arguments passed to application main method
      */
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
+    	switch (args.length) {
+    	case 0:
+    		setupDefaultFileForStorage();
+    		break;
+    	case 1: 
+    		setupGivenFileForStorage(args[0]);
+    		break;
+    	default:
```

Comment:
Good refactoring here. However, indentation is off. Our standard requires 4 spaces.